proposals/2025/Add Loop.md

```diff
@@ -0,0 +1,336 @@
+            </xs:element>
+            <xs:element name="FluidLoopDesignAndControl" minOccurs="0">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="DesignSupplyTemperature" minOccurs="0">
+                            <xs:annotation>
+                                <xs:documentation>Design Supply Temperature. (°C)</xs:documentation>
+                            </xs:annotation>
+                            <xs:complexType>
+                                <xs:simpleContent>
+                                    <xs:extension base="xs:decimal">
+                                        <xs:attribute ref="auc:Source"/>
+                                    </xs:extension>
+                                </xs:simpleContent>
+                            </xs:complexType>
+                        </xs:element>
+                        <xs:element name="DesignReturnTemperature" minOccurs="0">
+                            <xs:annotation>
+                                <xs:documentation>Design Return Temperature. (°C)</xs:documentation>
+                            </xs:annotation>
+                            <xs:complexType>
+                                <xs:simpleContent>
+                                    <xs:extension base="xs:decimal">
+                                        <xs:attribute ref="auc:Source"/>
+                                    </xs:extension>
+                                </xs:simpleContent>
+                            </xs:complexType>
+                        </xs:element>
+                        <xs:element name="MinimumFlowFraction" minOccurs="0">
+                            <xs:annotation>
+                                <xs:documentation>Minimum fraction of full flow allowed. (0-1)</xs:documentation>
+                            </xs:annotation>
+                            <xs:complexType>
+                                <xs:simpleContent>
+                                    <xs:extension base="xs:decimal">
+                                        <xs:attribute ref="auc:Source"/>
+                                    </xs:extension>
+                                </xs:simpleContent>
+                            </xs:complexType>
+                        </xs:element>
+                        <xs:element name="FluidLoopOperationTypes" minOccurs="0">
+                            <xs:annotation>
+                                <xs:documentation>Type of operation used by loop.</xs:documentation>
+                            </xs:annotation>
+                            <xs:simpleType>
+                                <xs:restriction base="xs:string">
+                                    <xs:enumeration value="Continuous"/>
+                                    <xs:enumeration value="Intermittent"/>
+                                    <xs:enumeration value="Demand"/>
+                                    <xs:enumeration value="Scheduled"/>
```

Comment:
Consider renaming this enumeration to "Timer" to match preexisting enumerations (e.g., https://github.com/BuildingSync/schema/blob/develop-v2/BuildingSync.xsd#L8075-L8089).